spec/services/cohort_service_spec.rb

```diff
@@ -23,12 +24,12 @@ def refresh_views
     # - 2 no BP in Q2
 
     q1_patients = [
-      create(:patient, registration_facility: facility, recorded_at: jan_5),
-      create(:patient, registration_facility: facility, recorded_at: jan_5 + 10.days),
-      create(:patient, registration_facility: facility, recorded_at: jan_5 + 20.days),
-      create(:patient, registration_facility: facility, recorded_at: jan_5 + 30.days),
-      create(:patient, registration_facility: facility, recorded_at: jan_5 + 45.days),
-      create(:patient, registration_facility: facility, recorded_at: jan_5 + 60.days)
+      create(:patient, registration_facility: facility, registration_user: user, recorded_at: jan_5),
+      create(:patient, registration_facility: facility, registration_user: user, recorded_at: jan_5 + 10.days),
+      create(:patient, registration_facility: facility, registration_user: user, recorded_at: jan_5 + 20.days),
+      create(:patient, registration_facility: facility, registration_user: user, recorded_at: jan_5 + 30.days),
+      create(:patient, registration_facility: facility, registration_user: user, recorded_at: jan_5 + 45.days),
+      create(:patient, registration_facility: facility, registration_user: user, recorded_at: jan_5 + 60.days)
```

Comment:
I wish we had an easier way to have large amounts of fixture data use the same associated objects ...saves a lot on setup time.

Reply:
Hmm, yeah. I can't find a FactoryBot way to do this. We _could_ pull the common attributes into a hash and splat-include it in each line here.

```ruby
associations = { registration_facility: facility, registration_user: user }

create(:patient, recorded_at: jan_5, **associations),
# ...
```

Ideally, we want some sort of `create_with() do` thing 🤔